### PR TITLE
added Entity layer

### PR DESCRIPTION
**Pull request by kozhukhovsky**

### CODE REVIEW

File: src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/Role.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package me.kozhukhovsky.bookingapartmentsystem.entity;
+
+public class Role {
```
Comment (sanyaivanov) [inline src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/Role.java]: I think "Authority" sounds better

File: src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+import java.util.Set;
+
+public class User {
+    private Long id;
+    private String name;
+    private String surname;
+    private Integer age;
+    private String phoneNumber;
+    private String login;
```
Comment (sanyaivanov) [inline src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java]: can be email

File: src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
+package me.kozhukhovsky.bookingapartmentsystem.entity;
+
+import java.util.Set;
+
+public class User {
+    private Long id;
+    private String name;
+    private String surname;
+    private Integer age;
+    private String phoneNumber;
```
Comment (sanyaivanov) [inline src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java]: can be phone

File: src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/Role.java (hunk 1 above)
Comment (kozhukhovsky) [reply src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/Role.java]: done

File: src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java (hunk 2 above)
Comment (kozhukhovsky) [reply src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java]: done

File: src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java (hunk 3 above)
Comment (kozhukhovsky) [reply src/main/java/me/kozhukhovsky/bookingapartmentsystem/entity/User.java]: done

**Pull request merged**